File: ITS.Domain/Entities/Transfer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ITS.Domain.Entities
{
    public class Transfer
    {
        public int TransferId { get; set; }
        public int InmateId { get; set; }
        public string InmateName { get; set; }
        public int SourceFacilityId { get; set; }
        public string SourceFacilityName { get; set; }
        public int DestinationFacilityId { get; set; }
        public string DestinationFacilityName { get; set; }
        public DateTime DepartureTime { get; set; }
        public DateTime ArrivalTime { get; set; }
    }
}

File: ITS.Web/src/app/domain/entities/transfer.ts
export class Transfer {
    constructor(
      public transferId: number,
      public inmateId: number,
      public inmateName: string,
      public sourceFacilityId: number,
      public sourceFacilityName: string,
      public destinationFacilityId: number,
      public destinationFacilityName: string,
      public departureTime: Date,
      public arrivalTime: Date
    ) {}
  }